### PR TITLE
Adding Gekkering for Vulps, Emot Keyword

### PR DESCRIPTION
**Pull request by neuPanda**
# Description
added key word for vulp emotes: gekker

:cl:
- add: gekker for vulps

### CODE REVIEW

File: Resources/Prototypes/Voice/speech_emotes.yml [hunk 1]
```diff
@@ -19,6 +19,13 @@
     - giggles
     - chortles
 
+- type: emote
+  id: Gekker
+  category: Vocal
+  chatMessages: [ laughs ]
```
Comment (Mnemotechnician) [inline Resources/Prototypes/Voice/speech_emotes.yml:25]:
```suggestion
  chatMessages: [ gekkers ]
```

**Comment by Mnemotechnician** [comment]
This also should use a different sound collection, but uhhh... we will need to find some sounds for that I suppose.

**Review by Memeji — CHANGES_REQUESTED** [review]
Give it its own sound / sound collection, or add "gekkers" under the "Laugh" emote chat triggers.

**Comment by neuPanda** [comment]
> Give it its own sound / sound collection, or add "gekkers" under the "Laugh" emote chat triggers.

adding it under the laugh emote triggers is it makes it available for all races if I am not mistaken. though it is possible I ended up with a tangled wire when trying to follow this

**Review by Memeji — APPROVED** [review]

**Pull request merged**